docs/src/css/custom.css

```diff
@@ -10,6 +10,8 @@ Copyright © 2021 Phylo, Dr. Electron, Jeroen van den Hout, Critical
 }
 
 :root {
+  --ifm-code-border-radius: 0;
```

Comment:
why?

Reply:
It looks terrible, and the rounder corners sometimes butt into the text